io.github.dosbox-staging.yml

```diff
@@ -0,0 +1,52 @@
+app-id: io.github.dosbox-staging
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+
+command: dosbox
+
+build_options:
+  cppflags: -DNDEBUG
+  cflags: -O3
+  cxxflags: -O3
```

Comment:
By default, flatpak-builder uses slightly longer list of flags:

```
{
  'libdir': '/usr/lib/x86_64-linux-gnu',
  'cflags': '-O2 -g -pipe -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection ',
  'cxxflags': '-O2 -g -pipe -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection ',
  'cppflags': '',
  'ldflags': '-L/app/lib -Wl,-z,relro,-z,now -Wl,--as-needed '
}
```

You probably want to keep some of these.

Reply:
Unless `cflags-override: true` is set, custom cflags are appended to default cflags, not replace them.

Reply:
Ah, that must be relatively new. It used to be all or nothing.

Reply:
I used `make V=1` flag to display all the flags that are passed to compilation and yes, all important ones mentioned above are there :) -O3 is appended after -O2 - I double-checked and GCC interprets the latter -O flag, so it works correctly as well :)